### PR TITLE
Bootstrap 5 template update

### PR DESCRIPTION
**Pull request by hardcopi**
Updated all the templates to Bootstrap 5. Tested it as best I can but there might be some issues here and there. Let me know.

### CODE REVIEW

File: server/templates/labelVideo.html [hunk 1]
```diff
@@ -16,144 +16,192 @@
...
-      href="https://fonts.googleapis.com/icon?family=Material+Icons">
-  <link rel="stylesheet"
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap">
-  <link rel="stylesheet" type="text/css"
-      href="https://storage.googleapis.com/{{project_id}}/static/css/styles.css?ignoreCache={{time_time}}">
-  <script type="text/javascript"
-      src="https://storage.googleapis.com/{{project_id}}/compiled/js/fmltc.js?ignoreCache={{time_time}}">
-  </script>
-  <title>Video Frame Labeling</title>
+  <title>FIRST Machine Learning Toolchain</title>
```
Comment (lizlooney) [inline server/templates/labelVideo.html:19]: @cmacfarl Do we want to put in a different placeholder title here? Maybe FIRST Object Detection Training Tool?

File: server/templates/login.html [hunk 2]
```diff
@@ -16,63 +16,89 @@
...
+				<label for="program">Program:</label>
+			</div>
+
+			<div class="form-floating">
+				<input type="text" class="form-control" name="team_number" id="teamNumber" placeholder="" value="{{team_number}}">
+				<label for="teamNumber">Team Number</label>
+			</div>
+			<div class="form-floating">
+				<input type="password" name="team_code" class="form-control" id="teamNumber" placeholder="">
+				<label for="teamNumber">Team Code</label>
```
Comment (lizlooney) [inline server/templates/login.html:94]: This should be ...label for="teamCode"...

File: server/templates/login.html [hunk 3]
```diff
@@ -16,63 +16,89 @@
...
+				</select>
+				<label for="program">Program:</label>
+			</div>
+
+			<div class="form-floating">
+				<input type="text" class="form-control" name="team_number" id="teamNumber" placeholder="" value="{{team_number}}">
+				<label for="teamNumber">Team Number</label>
+			</div>
+			<div class="form-floating">
+				<input type="password" name="team_code" class="form-control" id="teamNumber" placeholder="">
```
Comment (lizlooney) [inline server/templates/login.html:93]: This should be ...id="teamCode"...

**Review by lizlooney — CHANGES_REQUESTED** [review]

File: server/templates/labelVideo.html (hunk 1 above)
Comment (hardcopi) [reply server/templates/labelVideo.html:19]: I would prefer this. The initials alone of the current one are... questionable. FODTT while it doesn't roll of the tongue it doesn't invite memes. :)

File: server/templates/login.html (hunk 2 above)
Comment (hardcopi) [reply server/templates/login.html]: Oops, fixed in the latest push.

File: server/templates/login.html (hunk 3 above)
Comment (hardcopi) [reply server/templates/login.html]: Yeah... just hanging my head in shame here. Fixed though.

**Pull request merged**